### PR TITLE
Appzi nps style override 1

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

WIP

- Tries to address https://github.com/cowprotocol/cowswap/issues/862

Todo/need help:
- Could use some help with adding the unique class. We need to make sure the NPS widget is added. Haven't checked if the Appzi init function provide a function callback. Otherwise we'd need an async alternative. The goal is to add a unique (css) class with the NPS key to target with the override CSS styles.

### CODE REVIEW

**Comment by alfetopito** [comment]
Besides the adjustments in regards to the class still missing, when the styles are applied the modal is broken.
The input field is no longer visible, and some of the numbers become unclickable

https://user-images.githubusercontent.com/43217/182624493-db255982-c1e5-41f2-a9d5-ac4cfc640e08.mov

To make the testing easier locally, I've attached to [the header](https://github.com/cowprotocol/cowswap/blob/50596ae4ecdca849bce2ed832c68756b5b91df25/src/custom/components/Header/index.tsx#L70), the trigger to always show the NPS form
Screen Shot 2022-08-03 at 14 49 54

**Comment by anxolin** [comment]
Regarding Leandro's comment this is whats blocking us from clicking the buttons. U can only click the top part of them 
image

**Review by anxolin — APPROVED** [review]
Looks great!

File: src/custom/theme/baseTheme.tsx [hunk 1]
```diff
@@ -422,6 +422,109 @@ export const ThemedGlobalStyle = createGlobalStyle`
     z-index: 2147483004!important;
     display: block!important;
 
+    // Appzi NPS override =============================
+    &.appzi-nps-55872789-593b-4c6c-9e49-9b5c7693e90a {
```
Comment (anxolin) [inline src/custom/theme/baseTheme.tsx:426]: I will do a follow up PR just to not hardcode this

Comment (anxolin) [reply src/custom/theme/baseTheme.tsx:426]: https://github.com/cowprotocol/cowswap/pull/876

**Pull request merged**